descriptors/ModuleDescriptor-template.json

```diff
@@ -290,6 +290,16 @@
           ],
           "permissionsRequired": []
         },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/marccat/authority-record/fixed-field-display-value",
+          "modulePermissions": [
+            "configuration.all"
+          ],
+          "permissionsRequired": []
```

Comment:
no required permissions?  This means that the API can be called w/o logging in... Is that really a requirement?

Reply:
I have add the permissions for this API